src/modules/messageTemplates/i18n/pl_PL.root.json

```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Napisz własne szablony wiadomości w prywatnych rozmowach.",
+    "name": "Szablony wiadomości",
+    "settings": {
+        "name": "Nazwa",
+        "subject": "Podmiot",
+        "template": "Szablon (zmienne (zobacz wiki): {{Nazwa Użytkownika}}, {{dzisiaj[+n]}})",
```

Comment:
`{{Nazwa Użytkownika}}` needs to stay `{{username}}` and `{{dzisiaj[+n]}}` needs to stay `{{today[+n]}}`